projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -111,7 +86,7 @@ public static function get_gpt_completion( $content ) {
 			return $result;
 		}
 
-		$response = Client::wpcom_json_api_request_as_user(
+		$response = Client::wpcom_json_api_request_as_blog(
```

Comment:
No, we want to be able to pinpoint loggedin users for security purposes I think

Reply:
I followed those instructions: p1673510335525029/1673452156.937829-slack-CBG1CP4EN
But I am happy to revert...

Reply:
If you prefer to make this feature available to only Jetpack connected users (aka excluding admins that haven't connected their WP.com account) then you might take this into account in the permission checks (non WPCOM ones) as well (also the UI but this is out of this PR's scope).

In general, this decision also affects the permission checks so it would be nice to reach a decision before further reviewing.

cc @jeherve